### ISSUE TITLE
Configure severity per rule

### ISSUE THREAD

**Issue by rubenvandeven** [valentjn/languagetool-languageserver#95]
**Is your feature request related to a problem? Please describe.**
Generally, I prefer to have spelling errors at a 'warning' severity so there is a clear indication that I need to address them.
Also, I really enjoy LanguageTool's suggestions for e.g. passive voice, or hints about quotes, these errors can be distracting while writing. In particular because they appear as equally severe as spelling mistakes.

**Describe the solution you'd like**
This could be addressed by configuring a 'hint' severity for PASSIVE_VOICE, while having a 'warning' severity for other errors. I can imagine other users have other preferences. Ideally, one can map diagnostic severity levels to rule ids, e.g.

```json
{
    "EN_QUOTES": "information",
    "PASSIVE_VOICE": "hint"
}
```
This way, while writing, one can ignore particular suggestions until later, while immediately acting upon brainless spelling errors.

**Describe alternatives you've considered**
Keeping things as they are now, and learn to ignoring the suggestions ;-)

Either way, I really enjoy using language-tool using vscode-ltex, thank you for developing and maintaining these packages!

**Comment by valentjn** [comment]
Thanks for the suggestion, this is now implemented.

And thank you for the kind words (it means a lot to me!), but I can only take credit for vscode-ltex and ltex-ls. LanguageTool is not from me.

**Comment by rubenvandeven** [comment]
Wow, that's blazing fast. Many thanks!

While I'm aware that you "only" maintain ltex, I admire anyone able to maintain _any_ floss package -- it's easy to underestimate the effort invested in these projects. Keep up the great work!

**Comment by valentjn** [comment]
Feature released in 13.0.0 (vscode-ltex 11.0.0).
